src/pages/patientView/PatientViewGenePanelModal/styles.scss

```diff
@@ -0,0 +1,31 @@
+#patient-view-gene-panel {
+  table {
+    width: 100%;
+    margin-top: 25px;
+    
+    thead {
+      border-bottom: 1px #e5e5e5 solid;
+    }
+    
+    td {
+      vertical-align: top;
+      padding-top: 5px;
+    }
+  }
+  
+  .input-group-sm {
+    margin-left: 5px;
+  }
+  
+  h4.panel-name {
+    margin-bottom: 15px;
+  }
+}
+
+.patientview-modal {
```

Comment:
there could be many modals on patient view.   if you use css modules, you can just name this .modal